src/main/java/apollo/upload/MavenUploadThread.java

```diff
@@ -0,0 +1,24 @@
+package apollo.upload;
+
+import apollo.maven.MavenDeployer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.nio.file.Path;
+
+public class MavenUploadThread implements Runnable {
```

Comment:
consider to change the name of the class to MavenUploadAsync

Reply:
Done